claim_validate.go

```diff
@@ -0,0 +1,113 @@
+package jwt
+
+import (
+	"errors"
+	"sort"
+	"time"
+)
+
+type JwtPayloadClaimValidator func(tokenClaims *Claims) error
+
+func IssuerValidator(expectedIss string) JwtPayloadClaimValidator {
+	return func(tokenClaims *Claims) error {
+		tokenIss, ok := tokenClaims.String("iss")
+		if !ok {
```

Comment:
Why is it required? Without this check you can match against the empty string (`""`) to require absence of the issuer claim. ;-)